angular-ngrx-scss/src/app/repository/components/repo-header/repo-header.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="wrapper">
+  <div class="top">
+    <ng-container *ngIf="repo$ | async as repo">
```

Comment:
suggestion: Can this `ng-container` be moved up to wrap around the whole component here, or after the wrapper div? Since all the inner components use the same observable, would make this a bit cleaner and require less observable checking.

Reply:
moved to the div after the wrapper, you were right they are using the same observable